contexts/PaymentContext/src/Domain/CalculatingPaymentDelayCalculator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\PaymentContext\Domain;
+
+use DateInterval;
+use DateTime;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class CalculatingPaymentDelayCalculator implements PaymentDelayCalculator {
```

Comment:
Calculating payment delay calculator is calculating...

Yeah, I was kind of struggling with finding a name for it.